File: Linguaggio_Medio_Livello/C/Medium/reverse_string.c
#include <stdio.h>
#include <string.h>

int main(void) {
    char word[] = "programmazione";
    size_t length = strlen(word);

    for (size_t i = 0; i < length / 2; i++) {
        char temp = word[i];
        word[i] = word[length - 1 - i];
        word[length - 1 - i] = temp;
    }

    printf("Inverso: %s\n", word);
    return 0;
}

File: Linguaggio_Alto_Livello/Java/Medium/ReverseString.java
public class ReverseString {
    public static void main(String[] args) {
        String word = "programmazione";
        String reversed = new StringBuilder(word).reverse().toString();
        System.out.println("Originale: " + word);
        System.out.println("Inverso: " + reversed);
    }
}
